apps/stats-mainnet/.browserslistrc

```diff
@@ -0,0 +1,16 @@
+# This file is used by:
+# 1. autoprefixer to adjust CSS to support the below specified browsers
+# 2. babel preset-env to adjust included polyfills
+#
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+#
+# If you need to support different browsers in production, you may tweak the list below.
+
+last 1 Chrome version
```

Comment:
follow up (non-blocking): need a meeting to decide our browser support. THis does not include brave, which I would consider probably in our list of supported browsers. Probably will still work as it's backed onto Chrome V8 but something we should nail down as a team

Reply:
Good shout